eng/pipelines/common/templates/pipeline-with-resources.yml

```
@@ -79,9 +79,9 @@ extends:
         env:
           ROOTFS_DIR: /crossrootfs/x64
 
-      # We use a CentOS Stream 8 image here to test building from source on CentOS Stream 8.
+      # We use a CentOS Stream 8 image here to test building from source on CentOS Stream 9.
```

```suggestion
      # Test building from source on CentOS Stream.
```
Delete version numbers from the comment